integration-tests/smartAuth.test.ts

```diff
@@ -0,0 +1,50 @@
+import { AxiosInstance } from 'axios';
+import { getFhirClient, idsOfFhirResources } from './utils';
+
+const sherlockId = idsOfFhirResources.sherlockHolmes;
+const mycroftId = idsOfFhirResources.mycroftHolmes;
+
+async function getPatient(client: AxiosInstance, patientId: string | undefined = undefined) {
+    const url = patientId ? `/Patient/${patientId}` : '/Patient';
+
+    return client.get(url);
+}
```

Comment:
I don't like that this `getPatient` can be a READ or a SEARCH depending on the presence of `patientId`

looks like it is never used without a patientId parameter. Plus, if you wanted to search with this fn there's no way to pass search params. 

I'd rather remove the `patientId` param or maybe even remove this function altogether(It's fairly simple to just call `fhirClient.get('Patient/${sherlockId}');`)

Reply:
You're right. This function can be updated to remove the case in which no `patientId` is passed to it. This was left over for when I was using this function to also do searches. I still prefer to keep this function though, as I find it makes the code slightly more readable to have that function then to know to call `/Patient/${patientId}`